.travis.yml

```diff
@@ -16,13 +24,11 @@ rvm:
   - 2.2.1
   - 2.2.2
   - 2.2.3
+  - jruby-9000
```

Comment:
I eas using 9.0.1.0 on my machine, but when I wrote this PR rvm wasn't with this version available :/